### ISSUE TITLE
possible bug that shows up with using covariant errors

### ISSUE THREAD

**Issue by eclake** [jmeyers314/linmix#18]
Hi,

Thank you so much for posting your version of this program!  I've been playing a little and was getting non-sensical results when I ran with covarying errors, and tracked it down to [these lines](https://github.com/jmeyers314/linmix/blob/34b312d407b3570dc1cfab57d5cd092bd4476487/linmix/linmix.py#L171-L175)

I think they should read
```
        xihat_ik = (sigma_xihat_i_sqr[:, np.newaxis]
                    * ((xihat_xy_i/(self.xvar
                        * (1.0 - self.xycorr**2)))[:, np.newaxis]
                       + self.beta*(self.eta[:, np.newaxis] - self.alpha)/self.sigsqr
                       + self.mu/self.tausqr))
```
instead of:
```
        xihat_ik = (sigma_xihat_i_sqr[:, np.newaxis]
                    * ((xihat_xy_i/self.xvar
                        * (1.0 - self.xycorr**2))[:, np.newaxis]
                       + self.beta*(self.eta[:, np.newaxis] - self.alpha)/self.sigsqr
                       + self.mu/self.tausqr))
```

**Comment by jmeyers314** [comment]
Yep.  That definitely looks like a bug.  I pushed your suggested fix to master: [933dbb1](https://github.com/jmeyers314/linmix/commit/933dbb1359dcb5404cf881bfdd5cf433b0195152).  If you're happy, then please feel free to close this issue.

**Comment by eclake** [comment]
Looks good :) - Thanks!
